.travis.yml

```diff
@@ -14,5 +18,8 @@ install:
 
 script:
   - export PYTHONPATH=`python -c "import sys; print('python2' if sys.version.startswith('2') else 'src')"`; py.test $PYTHONPATH
+  - python setup.py install;
```

Comment:
The problem is that the `typing_extensions` depends on `typing` being installed for Python 3.4 and below since we try and re-export as much as possible from the `typing` module. That said, I've tried adjusting travis so it should install `typing` only for Python 3.4 and below -- we'll see if I did it correctly once the tests are done.